app/views/passengers/show.haml

```diff
@@ -11,15 +9,9 @@
   %b Phone:
   = @passenger.phone
 %p
-  %b Wheelchair:
-  - if @passenger.wheelchair
-    Yes
-  - else
-    No
-%p
   %b Mobility Device:
-  - if @passenger.mobility_device?
-    = @passenger.mobility_device
+  - if @passenger.try(:mobility_device)
+    = @passenger.mobility_device.device
```

Comment:
Just `passenger.mobility_device.try(:device)`, with no conditional, will work.

If there is a mobility device, then `try` calls `#device` on it. If there is not (it is `nil`) then it just returns `nil`.

Reply:
It will just leave it blank on the page if there is no mobility device instead of showing 'None'. Is that fine?

Reply:
Nvm, I remembered a way to work on that

`= @passenger.mobility_device.try(:device) || 'None'`